ci/pipeline

```diff
@@ -278,11 +276,12 @@ def release(requestVersion: String, gitSha: String, latest: Boolean = false): Un
     println("Docker image mesosphere/marathon:latest NOT updated")
   }
 
+  %('sbt, "publish")
```

Comment:
Let's put a comment here saying why we publish? I'm afraid future travelers will miss the connection.